core/src/Platforms/iOS/EmbeddedWebview/EmbeddedWebUI.cs

```diff
@@ -65,29 +54,34 @@ public static void SetAuthorizationResult(AuthorizationResult authorizationResul
 
         public void Authenticate(Uri authorizationUri, Uri redirectUri, RequestContext requestContext)
         {
+            UIViewController vc = null;
```

Comment:
>vc [](start = 29, length = 2)

nit: please use viewController or similar instead of abbriviating it. it will imo be easier to read the code

Reply:
done.